internal/driver/glfw/canvas.go

```diff
@@ -361,8 +361,7 @@ func (c *glCanvas) paint(size fyne.Size) {
 
 	paint := func(node *renderCacheNode, pos fyne.Position) {
 		obj := node.obj
-		// TODO should this be somehow not scroll container specific?
-		if _, ok := obj.(*widget.ScrollContainer); ok {
+		if _, ok := obj.(fyne.Scrollable); ok {
```

Comment:
`fyne.Scrollable` does not mean that this is a container with clipped and scrollable content but that this is an object which handles scroll events.
For a number entry with up/down arrows on the right `fyne.Scrollable` might provide a way to increase/decrease using the mouse.

Reply:
Good point, thanks